### PR TITLE
Removing services from ignore list

### PR DESCRIPTION
**Pull request by atharvar28**
removing svcs from `svcs_to_ignore` list

### CODE REVIEW

**Review by themarcelor — APPROVED** [review]
+1

File: gen3release-sdk/gen3release/config/env.py [hunk 1]
```diff
@@ -159,6 +157,8 @@ def _replace_all_versions(self, version, override, json_block):
             logging.debug("Malformed override json string passed - {}".format(override))
             dict_override = {}
         for svc in json_block:
+            if "dashboard" not in json_block:
```
Comment (haraprasadj) [inline gen3release-sdk/gen3release/config/env.py:160]: should we add this check for audit-service too? not sure if it is in all the commons yet

**Review by haraprasadj — APPROVED** [review]
+1

File: gen3release-sdk/gen3release/config/env.py (hunk 1 above)
Comment (atharvar28) [reply gen3release-sdk/gen3release/config/env.py:160]: I believe some of the commons do have audit-service deployed to them, but I am guessing all the envs related to that particular commons would use audit service. But for `dashboard`, some of the preprod envs dont use them. so when we do code promotion, `dashboard` is removed while making the PR to cdis-manifest.

**Review by haraprasadj — APPROVED** [review]

**Review by themarcelor — APPROVED** [review]
+1

**Pull request merged**